backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +687,128 @@ def test_cancel_workshop_attendance(self):
+        self.time_one = self.project_1.start_time - timedelta(days=6)
+        self.time_two = self.project_1.start_time + timedelta(days=4)
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_correct_ids_returned_before_or_after_review_request(self):
+
+        # Four reviews are made with the four review times above (No reviews done on project_2)
+        review_1 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=self.time_one
+        )
+        review_1.timestamp = self.time_one
+        review_1.save()
+
+        review_2 = factories.RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = self.time_two
+        review_2.save()
+
+        review_3 = factories.RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = self.time_three
+        review_3.save()
+
+        review_4 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_4.timestamp = self.time_four
+        review_4.save()
+
+        ids_which_should_be_returned = [review_2.reviewer_user.id, review_3.reviewer_user.id, review_4.reviewer_user.id]
+
+        # Making sure that reviews were done on card and not on card_2, if card_2 returns reviews then our function is
+        # returning the wrong stuff and therefore it is not working as it should.
+        self.assertEqual(
+            sorted(self.card_1.get_users_that_reviewed_since_last_review_request()),
+            sorted(ids_which_should_be_returned)
+        )
+
+    def test_no_user_ids_should_return_since_review_done_before_latest_review_request(self):
+
+        # Create a review request on card_2 of project_2
+        request_review_time_2 = self.project_2.start_time + timedelta(1)
+        self.project_2.request_review(force_timestamp=request_review_time_2)
+        time_of_review = self.project_2.start_time - timedelta(days=10)
+
+        # Create a review on card_2, the review will be before the latest request for a review
+        review_1_card_2 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_2,
+            timestamp=time_of_review
+        )
+        review_1_card_2.timestamp = time_of_review
+        review_1_card_2.save()
+
+        # Review done on project_2, card_2 were done before the latest request for a review, therefore,
+        # get_users_that_reviewed_since_last_review_request() should return no user id's
+        assert len(self.card_2.get_users_that_reviewed_since_last_review_request()) == 0
```

Comment:
why did you choose to use assert instead of `self.assertEqual`?

We are using the unittest framework, it has certain conventions. Those conventions lead to nice logging of errors.

Eg if this assertion were to fail as it was then the traceback would say something like this: 

```
Traceback (most recent call last):
  File "/home/sheena/workspace/Tilde/backend/curriculum_tracking/tests/test_AgileCard.py", line 790, in test_no_user_ids_should_return_since_review_done_before_latest_review_request
    assert stuff
AssertionError
```
As you can see, that is not super informative.

If you replaced your assert with self.assertEqual then the error would look like this:

```
Traceback (most recent call last):
  File "/home/sheena/workspace/Tilde/backend/curriculum_tracking/tests/test_AgileCard.py", line 790, in test_no_user_ids_should_return_since_review_done_before_latest_review_request
    self.assertEquals(... things)
AssertionError: 1 != 2     # isn't that nice?
```

Can you please switch over to use the convenions in place instead of using standard assert statements